lms/services/event.py

```diff
@@ -31,7 +31,7 @@ def insert_event(self, event: BaseEvent):
         self._db.add(db_event)
 
         if event.user_id:
-            for role_id in event.role_ids or [None]:
+            for role_id in event.role_ids or [None]:  # type: ignore
```

Comment:
This is arguably a bit of a wart, we want to insert one per role id or just one if no roles are present.

Happy to ignore it anyway.